README.md

```diff
@@ -260,23 +260,23 @@ $ peg install <cluster-name> <technology>
 ```
 The `technology` tag can be any of the following:
 * alluxio (v1.0.0)
-* cassandra (v3.6)
+* cassandra (v3.7)
 * elasticsearch (v2.3.3)
-* flink (v1.0.0 with hadoop v2.7 and scala v2.10)
+* flink (v1.1.2 with hadoop v2.7 and scala v2.10)
 * hadoop (v2.7.2)
 * hbase (v1.2.1)
 * hive (v1.2.1)
-* kafka (v0.9.0.1 with scala v2.10)
-* kafka-manager (v1.3.0.8)
+* kafka (v0.10.0.0 with scala v2.10)
+* kafka-manager (v1.3.1.8+)
```

Comment:
@rheeluke It seems it is v1.3.1.8. Mind updating? I checked it here. https://github.com/yahoo/kafka-manager/blob/master/build.sbt